lib/airtable/entity/base.rb

```diff
@@ -12,7 +12,7 @@ def table(name)
       end
 
       def __make_request__(method, path, data)
-        url  = [::Airtable.server_url, @id, path].join('/')
+        url  = [::Airtable.server_url, CGI.escape(@id), path].join('/')
```

Comment:
`@id` shouldn't need escaping for now. It can only be 17-character alphanumeric starting with "app".